package.json

```diff
@@ -23,18 +23,18 @@
 	],
 	"scripts": {
 		"build": "tsup",
-		"create:test": "node script/create-test-e2e.js",
+		"test:create": "node script/create-test-e2e.js",
 		"format": "prettier \"**/*\" --ignore-unknown",
 		"format:write": "pnpm format --write",
 		"initialize": "tsx ./src/bin/index.js --mode initialize",
-		"initialize:test": "node script/initialize-test-e2e.js",
+		"test:initialize": "node script/initialize-test-e2e.js",
 		"lint": "eslint . .*js --max-warnings 0",
 		"lint:knip": "knip",
 		"lint:md": "markdownlint \"**/*.md\" \".github/**/*.md\" --rules sentences-per-line",
 		"lint:package-json": "npmPkgJsonLint .",
 		"lint:packages": "pnpm dedupe --check",
 		"lint:spelling": "cspell \"**\" \".github/**/*\"",
-		"migrate:test": "node script/migrate-test-e2e.js",
+		"test:migrate": "node script/migrate-test-e2e.js",
```

Comment:
Ah, forgot to mention, I like keeping these scripts sorted alphabetically 😄. Grouping them by name is helpful for scanning through & seeing all the options. And it helps searchability.

When I try this locally, the formatter does it... I'm surprised this isn't failing any CI tasks?

🤷 I can do it since I forgot to ask before